cpu/atmega2560/include/periph_cpu.h

```diff
@@ -20,12 +20,33 @@
 #define PERIPH_CPU_H_
 
 #include "periph/dev_enums.h"
+#include <avr/io.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-/* nothing defined here so far... */
+/**
+ * @brief   Define a CPU specific GPIO pin generator macro
+ */
+#define GPIO(x, y)          ((x << 4) | y)
+
+/**
+ * @brief   Available ports on the ATmega2560 family
+ */
+enum {
+    PORT_A = 0,       /**< port A */
+    PORT_B = 1,       /**< port B */
+    PORT_C = 2,       /**< port C */
+    PORT_D = 3,       /**< port D */
+    PORT_E = 4,       /**< port E */
+    PORT_F = 5,       /**< port F */
+    PORT_G = 6,       /**< port G */
+    PORT_H = 0,       /**< port H */
```

Comment:
@kaspar030 Hmm yes, I just used this for the calculation of the port address. But I found a little bug, so I changed it back to PORT_H = 7, PORT_J = 8 and so on.
Now I'm using an Offset to Port_H instead of a Base address, when the Port number is higher then PORT_G. So all pins are accessible and the 'ugly' thing in the periph_cpu.h is gone :-)